packages/iov-ethereum/README.md

```diff
@@ -0,0 +1,36 @@
+# @iov/ethereum
+
+[![npm version](https://img.shields.io/npm/v/@iov/ethereum.svg)](https://www.npmjs.com/package/@iov/ethereum)
+
+## Getting started
+
+The primary way to use @iov/ethereum is together with @iov/core. Alternatively,
+you can use @iov/ethereum to create offline transactions which can be posted manually.
+
+All examples are made for use in @iov/cli and you may need to include some
+missing symbols if used in a different environment.
+
+### Using with @iov/core
+
+You can use @iov/ethereum as an extension of @iov/core to interact with the
+Ethereum blockchain as follows.
+
+### Local test
+
+To run Ethereum tests locally, you need to install ganache and set as follow:
+- RPC Server: `http://127.0.0.1:7545`
+- Network ID: `5777`
+- Mnemonic: `oxygen fall sure lava energy veteran enroll frown question detail include maximum`
+
+Or run the followin script:
+```
+cd packages/iov-ethereum
+source startGanache.sh
```

Comment:
Using source for scripts is very problematic. I know we use it for BOV and tendermint, but that is only because we set environment variables that we need later. Problems with source:
- you run in the shell of the users, not the one you set after `#!`. So e.g. I cannot use bash features in script because I use zsh
- you cannot stop the script in case of an error in one command, because `exit 1` does not end the script but your current console

In https://github.com/iov-one/iov-core/pull/432/files#diff-a09db41a73f7fd39327cf6a25e928626R45 I run a script and set the env variable explicitly, which is better. In this case, we don't need env variables as far as I can see because the stop script uses the process ID from the process list.